tg/controllers.py

```diff
@@ -76,29 +76,25 @@ def __init__(self, model: Model, view: View, tg: Tdlib) -> None:
         self.tg = tg
         self.chat_size = 0.5
 
+    @bind(msg_handler, ["c"])
```

Comment:
Why does the shortcut attached to  the `msg_handler`? I believe it should be in `chat_handlers`. 
We can use  `i`(stands from _information_) shortcut  for chat panel.

Reply:
I think it should be in msg handler, because this is what follows official client logic. When you open chat and at the top there is a chat title, when you tap you see more info.